panda/src/collide/collisionHeightfield.I

```diff
@@ -0,0 +1,54 @@
+INLINE CollisionHeightfield::
```

Comment:
There's no file header in this file.  Please follow the example of other .I files.

Similarly, please add a docstring comment above each method definition (even if it's empty at first) of the form:
```cpp
/**
 *
 */
```